en/docs/setup/deployment-Patterns.md

```diff
@@ -2,68 +2,59 @@
 patterns take [high
-availability](https://docs.wso2.com/display/ADMIN44x/Clustering+Overview)
+availability](../../admin-guide/clustering-overview)
 into consideration and are recommended for production deployment
 environments. The following sections provide high level information on
 the recommended patterns available and point off to instructions on how
-to set up and configure the deployment pattern.
-
-------------------------------------------------------------------------
-
--   [Deployment
-    prerequisites](#DeploymentPatterns-Deploymentprerequisites)
--   [Deployment Patterns](#DeploymentPatterns-DeploymentPatterns)
-    -   [Pattern 1 - HA clustered deployment of WSO2 Identity
-        Server](#DeploymentPatterns-Pattern1-HAclustereddeploymentofWSO2IdentityServer)
-    -   [Pattern 2 - HA clustered deployment of WSO2 Identity Server
-        with WSO2 Identity
-        Analytics](#DeploymentPatterns-Pattern2-HAclustereddeploymentofWSO2IdentityServerwithWSO2IdentityAnalytics)
-
-------------------------------------------------------------------------
+to set up and configure the deployment pattern. 
 
 ### Deployment prerequisites
 
 As a first step in planning your deployment, ensure that you have the
 necessary system requirements and a compatible environment.
 
-**System requirements**
-
-|        |                                                                                          |
-|--------|------------------------------------------------------------------------------------------|
-| Memory | **4GB** : 2 GB for the Java Virtual Machine (JVM) and 2 GB for the Operating System (OS) |
-| Disk   | 10 GB minimum                                                                            |
-
-**Environment compatibility**
-
+**System requirements** 
+<table>
+    <tr>
+        <th>Memory</th>            
+        <td><b>4GB</b>: 2 GB for the Java Virtual Machine (JVM) and 2 GB for the Operating System (OS)</td>
+    </tr>
+    <tr>
+        <th>Disk</th>            
+        <td>10 GB minimum</td>
+    </tr>
+</table>                                                                        |
+
+**Environment compatibility** 
 <table>
 <tbody>
 <tr class="odd">
-<td>Operating systems</td>
-<td><p>For information on tested operating systems, see <a href="https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs">Tested Operating Systems and JDKs</a> .</p></td>
+<th>Operating systems</th>
+<td><p>For information on tested operating systems, see <a href="../../setup/environment-compatibility#tested-operating-systems-and-jdks">Tested Operating Systems and JDKs</a> .</p></td>
 </tr>
 <tr class="even">
-<td>RDBMS</td>
-<td><p>For information on tested DBMSs, see <a href="https://docs.wso2.com/display/compatibility/Tested+DBMSs">Tested DBMSs</a> .</p></td>
+<th>RDBMS</td>
+<td><p>For information on tested DBMSs, see <a href="../../setup/environment-compatibility#tested+dbmss">Tested DBMSs</a> .</p></td>
```

Comment:
Please change the plus sign to a hyphen in #tested+dbmss

Reply:
Updated